lib/lotus/model/migrator.rb

```diff
@@ -314,7 +314,7 @@ def self.migrations
       # @since 0.4.0
       # @api private
       def self.migrations?
-        migrations.children.any?
+        migrations.children.reject { |file| file.basename.to_s.start_with?('.') }.any?
```

Comment:
Can we use `detect`, it will avoid running through all migrations? :)

```ruby
 !migrations.children.detect { |file| file.basename.to_s.start_with?('.') }.nil?
```